behat.yml

```diff
@@ -47,18 +47,22 @@ default:
     # Show explicit fail information and continue the test run.
     DrevOps\BehatFormatProgressFail\FormatExtension: ~
 
-p0:
+default0:
```

Comment:
This changed from p0 to default0 - is that right?

Reply:
correct

Since we needed to add another dimension to the testing matrix (with and without sub-theme), the logic to produce the name of the Behat profile while running in the parallel containers has changed (https://github.com/salsadigitalauorg/civictheme_source/pull/876/files#diff-9df30662fc907fc1eed876097154da3b7830a655f8595923a81f38420b084162R110): we are now appending index to the profile passed from the CI job rather then completely replacing it.

Before this change: 
If `DREVOPS_TEST_BEHAT_PROFILE=myprofile` and parallelism is enabled in CI - the resulting name will be `p` + `<index of CI container>` (e.g. `p0`, `p1` etc).

After this change: 
If `DREVOPS_TEST_BEHAT_PROFILE=myprofile` and parallelism is enabled in CI - the resulting name will be `<DREVOPS_TEST_BEHAT_PROFILE>` + `<index of CI container>` (e.g. `myprofile0`, `myprofile1` etc). 

If `DREVOPS_TEST_BEHAT_PROFILE=<unset>` and parallelism is enabled in CI - the resulting name will be `default` + `<index of CI container>` (e.g. `default0`, `default1` etc).

Reply:
Thanks this now makes perfect sense